client/landing/stepper/declarative-flow/free.ts

```diff
@@ -93,6 +106,15 @@ const free: Flow = {
 						);
 					}
 
+					if ( isAssemblerDesign( selectedDesign ) ) {
```

Comment:
It might be better to handle the processing result here as well. I'm not sure why it didn't deal with it before 😓
```js
const processingResult = params[ 0 ] as ProcessingResult;

if ( processingResult === ProcessingResult.FAILURE ) {
	return navigate( 'error' );
}
```

It's not a blocker since this is out of the scope.

Reply:
✅ Done in the follow-up PR.